e2e/cypress/support/api/system.js

```diff
@@ -152,3 +152,16 @@ function uploadLicenseIfNotExist() {
         });
     });
 }
+
+Cypress.Commands.add('apiDeleteBrandImage', () => {
```

Comment:
I wonder these should be included into UI tests. I mean, it seems like we are skipping the system console test here.

Reply:
I added it like a function as it was more a precondition of the test than an action (make sure no image has been uploaded yet VS remove the uploaded image)

Reply:
Thanks for bringing up @isacikgoz but I agree with @mgdelacroix. We'll cover such case in [MM-T1030](https://automation-test-cases.vercel.app/test/MM-T1030).

Reply:
Sounds good.